docs/quickstarts/ratify-cli.md

```diff
@@ -0,0 +1,119 @@
+# Ratify Cli
+
+This document provides a walkthrough on how to use the ratify command line to verify a sample image.
+
+## Verify notation signature with Ratify cli
+
+1. Download the latest Ratify cli
+```bash
+curl -L https://github.com/ratify-project/ratify/releases/download/v1.2.1/ratify_1.2.1_Linux_amd64.tar.gz | tar xvzC ~/bin/ ratify
+```
+
+2. Download a local verification certificate for sample image
+```bash
+curl -sSLO https://raw.githubusercontent.com/deislabs/ratify/main/test/testdata/notation.crt
```

Comment:
```suggestion
curl -sSLO https://github.com/ratify-project/ratify/blob/v1.2.1/test/testdata/notation.crt
```

Reply:
The documentation is versioned. Should we specify a version for the test certificate?  The test cert in `main` branch might be expired or revoked in the future.